CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * update MSAL test app for SSO Seeding flow #1021
 * update new variable in configuration to allow user by pass URI check #1013
 * Refactor crypto code for cpp integration and add api to generate ephemeral asymmetric key pair. #1018
+* Include redirect uri in body when redeeming refresh token at token endpoint (#1020)
```

Comment:
NIT: maybe move this to the top if it's a latest feature change

Reply:
addressed :)